src/Table/Table.jsx

```diff
@@ -98,19 +148,26 @@ Table.propTypes = {
   data: PropTypes.array.isRequired,
   filter: PropTypes.func,
   fixed: PropTypes.bool,
+  initialPage: PropTypes.number,
   initialSortState: tablePropTypes.sortState,
+  onPageChange: PropTypes.func,
   onSortChange: PropTypes.func,
+  pageSize: PropTypes.number,
+  paginated: PropTypes.bool,
   rowIDFn: PropTypes.func.isRequired,
 };
 
 Table.defaultProps = {
   filter: () => true,
+  onPageChange: () => {},
   onSortChange: () => {},
+  pageSize: DEFAULT_PAGE_SIZE,
 };
 
 Table.cssClass = {
   BODY: "Table--body",
   FIXED: "Table--fixed",
+  NO_DATA: "Table--no_data_cell",
```

Comment:
hm, seen many different formats over time, but usually dashes delineate hierarchy or modification (like table-row-cell) whereas snake_case or camelCase is used for logical units/compound names (like table-headerCell or table-header_cell)

do we have a pattern in place here? would be down to do a pass over this repo and fix all of them once we've settled on one